docs/Introduction/architecture-overview/architecture-decentralized-model.md

```diff
@@ -92,12 +92,12 @@ Learn how to create a consumer contract to [Get the Latest Price](../get-the-lat
 
 Proxy contracts are on-chain proxies that store the most up-to-date Aggregator for a particular price feed. Using proxies enables the underlying Aggregator to be upgraded without any interruption of service for consuming contracts.
 
-See the <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
+See the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
 
 ## Aggregators
 
 Aggregators are the contracts that receive periodic price updates from multiple Oracles. They aggregate and store the current price on-chain so that consumers can obtain the latest price and act upon it within the same transaction.
 
-This data can be accessed by referencing the Price Feed address using the <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> contract.
+This data can be accessed by referencing the Price Feed address using the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> contract.
```

Comment:
0.6 or 0.7 or 0.8?

Reply:
For now, it should be 0.6. I'll move it to 0.8 within the PR that moves code samples to 0.8.
But agree, this one should be 0.6. Thanks